ethereum_history_api/circuits/lib/src/merkle_patricia_proofs/proof.nr

```diff
@@ -32,7 +32,7 @@ struct ProofInput<MAX_KEY_LEN, MAX_VALUE_LEN, MAX_DEPTH, MAX_LEAF_LEN> {
 }
 
 pub fn get_node_len<MAX_LEN>(node: [u8; MAX_LEN]) -> u64 {
-    let node_rlp_header = decode_to_rlp_header(node);
+    let node_rlp_header = decode_to_rlp_header(Fragment::new_focused(node));
```

Comment:
Can we try implementing `From<[u8; N> for Fragment<u8; N>`? Seems like it would be very useful in this PR

Reply:
In the next PR

Reply:
Will do